### PR TITLE
Article: add and respect "is_published" field

### PR DESCRIPTION
**Pull request by mikek**

### CODE REVIEW

File: aldryn_newsblog/migrations/0006_article_adjust_verbose_names.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```
Comment (intellectronica) [inline aldryn_newsblog/migrations/0006_article_adjust_verbose_names.py]: This appears unrelated to this branch.

**Comment by intellectronica** [comment]
:+1:

File: aldryn_newsblog/migrations/0006_article_adjust_verbose_names.py (hunk 1 above)
Comment (mikek) [inline aldryn_newsblog/migrations/0006_article_adjust_verbose_names.py]: I just had to include these migrations as we do with general "typo" fixes. Django 1.7 considers even `verbose_name` changes a thing to put into migrations.

Comment (intellectronica) [inline aldryn_newsblog/migrations/0006_article_adjust_verbose_names.py]: Ah OK.
